weave/trace/weave_client.py

```diff
@@ -1345,7 +1354,7 @@ def send_start_call() -> bool:
                     started_at=started_at,
                     parent_id=parent_id,
                     inputs=inputs_json,
-                    attributes=attributes_dict,
+                    attributes=attributes_dict.unwrap(),
```

Comment:
Why do you need this?

Reply:
ugg - reconstruction of the attributes over the queue results in the `weave` key throwing an error b/c it thinks it is getting edited.